MilitaryManager.Attachments.API/StoreConfig/StoreModeConfig.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using MilitaryManager.Attachments.API.Services.StoreService;
+
+
```

Comment:
Move this file to Infrastructure. Meybe it`s better because this logic for work with global services. Need discuss